pkg/composer/parse.go

```diff
@@ -33,3 +36,25 @@ func Parse(r io.Reader) ([]types.Library, error) {
 	}
 	return libs, nil
 }
+
+func ParseWordPress(r io.Reader) ([]types.Library, error) {
```

Comment:
If multiple versions are not allowed, we should return a single object.
```suggestion
func ParseWordPress(r io.Reader) (types.Library, error) {
```